geo/src/lib.rs

```diff
@@ -1,43 +1,139 @@
 #![doc(html_logo_url = "https://raw.githubusercontent.com/georust/meta/master/logo/logo.png")]
-//! The `geo` crate provides geospatial primitive types such as `Coordinate`, `Point`, `LineString`, and `Polygon` as
-//! well as their `Multi–` equivalents, and provides algorithms and operations such as:
-//!   - Area and centroid calculation
-//!   - Simplification and convex hull operations
-//!   - Distance measurement
-//!   - Intersection checks
-//!   - Affine transforms such as rotation and translation.
-//!
-//! The primitive types also provide the basis for other functionality in the `Geo` ecosystem, including:
-//!   - Serialization to and from [GeoJSON](https://docs.rs/geojson) and [WKT](https://docs.rs/wkt)
-//!   - [Coordinate transformation and projection](https://docs.rs/proj)
-//!   - [Geocoding](https://docs.rs/geocoding)
-//!   - [Working with GPS data](https://docs.rs/gpx)
-//!
-//! …allowing these crates to interoperate; GeoJSON can readily be read from a file, deserialised, transformed
-//! to a local datum, modified, transformed back to `WGS84`, and serialised back to GeoJSON.
-//!
-//! Operations available for primitive types can be found in the `algorithm` module, along with
-//! comprehensive usage examples.
-//!
-//! While `Geo` is primarily intended to operate on **planar** geometries, some other useful algorithms are
-//! provided: Haversine, Frechet, and Vincenty distances, as well as Chamberlain-Duquette area.
-//!
-//! ## Optional Features (these can be activated in your `cargo.toml`)
-//! The following optional features are available:
-//! - `use-proj`: enable coordinate conversion and transformation of `Point` geometries using the [`proj`](https://docs.rs/proj) crate
-//! - `proj-network`: enables functionality for `proj` crate's network grid. After enabling
-//! this feature, some [further
-//! configuration](https://docs.rs/proj/0.20.5/proj/#grid-file-download) is
-//! required to actually use the network grid.
-//! - `use-serde`: enable serialisation of geometries using `serde`.
-//!
-//! ## GeoJSON
```

Comment:
Should we keep the GeoJSON section?

Reply:
I'd wager almost everyone will want to use geo to operate on data derived outside of geo, so I think it's worth including and also simultaneously introducing the context of the `geo-types` crate.

Here's some off-the-cuff copy:

> [geojson](https://crates.io/crates/geojson), [wkt](https://crates.io/crates/wkt), [shapefile](https://crates.io/crates/shapefile), and [other crates](#link to addendum with exhaustive list) which support `geo-types` allow you to use `geo` with your favorite spatial format (insert joke about shapefile).
> ...
> ## Formats that support geo-types
> - postgis
> - flatgeobuffer
> - any others??? 
It's a little tricky because it'd be nice to just gloss over the fact that geo-types exists for users of geo, but in reality, people using these formats may need to build the crate with `--feature geo-types` so it'd be good to prime them to look for the word `geo-types` in that crate's documentation.

Reply:
> I'd wager almost everyone will want to use geo to operate on data derived outside of geo

This was my rationale for the original inclusion of the GeoJSON section. I think it's useful to try to explicitly point out the various components of the Geo ecosystem to people as early as possible rather than relying on them to figure it out using the context from the docs.